src/Spice86.Shared/Utils/MemoryUtils.cs

```diff
@@ -27,4 +27,26 @@ public static uint ToPhysicalAddress(ushort segment, ushort offset) {
     public static ushort ToSegment(uint physicalAddress) {
         return (ushort)(physicalAddress >> 4);
     }
+
+    /// <summary>
+    /// Converts a physical address to a corresponding <see cref="SegmentedAddress"/>
+    /// </summary>
+    /// <param name="physicalAddress">The unsigned integer representing the physical memory address.</param>
+    /// <remarks>The maximum is equal to <c>0x10FFEF</c>, which is the end of the High Memory Area.</remarks>
+    /// <returns>A segmented address representation of the physical address.</returns>
+    public static SegmentedAddress ToSegmentedAddress([Range(0, 0x10FFEF)] uint physicalAddress) {
```

Comment:
I have mixed feelings about this. One the one hand it's nice to have consistent formats everyhwere, but on the other hand I don't think we should just make up segments that are never actually used by the game.

Reply:
I agree. And in the long run, cfgcpu is registering actual segmented addresses for all instructions (and handling case where an instruction is called with a different CS), so we will have a way of knowing the actual correct value(s)

Reply:
I can move what is displayed and the internal format of all ViewModels to be a physical address. That's easy to do.

This will remove the method above, along with its unit test.

Reply:
I think that's better for when there is no way of guessing the actual segmented address which anyways depends on the instruction being executed

Reply:
Done. We display hexadecimal (unless it's a CPU instruction in DISASM window, in this case segmented:address displayed is based on CS:IP), but accept hex, decimal, or segmented as input in textboxes.